custom_components/fire_protection_hu/sensor.py

```diff
@@ -71,14 +71,16 @@ def __init__(self, hass, name, county_id):
         self._name = name
         self._county_id = county_id
         self._state = None
-        self._fdata = [None] * 21
+        self._fdata = [None] * 20
         self._icon = DEFAULT_ICON
         self._session = async_get_clientsession(hass)
 
     @property
     def extra_state_attributes(self):
         attr = {}
         attr["provider"] = CONF_ATTRIBUTION
+        attr["county_id"] = int(self._county_id)
```

Comment:
What's the point adding the county_id as attribute? It's just an index. What's the use case for it?